NotesRepository/Services/Azure/AzureStorageHelper.cs

```diff
@@ -133,5 +133,11 @@ public async Task DeleteImageFromAzure(string imageName, string containerName)
             var container = OpenContianer(containerName);
             await container.DeleteBlobAsync(imageName);
         }
+
+        public void DeleteImageFromAzureNotAsync(string imageName, string containerName)
```

Comment:
There is no need to include `NotAsync` in method name in case, when the method is synchronus. In general, the convention is to add `Async` to the method name, when the method is `async`, but when it's not - the method does not include the `Async` word at the end of the method name.